src/Arrows.js

```diff
@@ -1,68 +1,105 @@
-import React, { Component, PropTypes } from 'react';
+import { default as React, Component, PropTypes } from 'react';
 
 class PrevArrow extends Component {
   static propTypes = {
+    className: PropTypes.string,
     handleClick: PropTypes.func,
     currentSlide: PropTypes.number,
     activeClassName: PropTypes.string,
     inactiveClassName: PropTypes.string,
-    infinite: PropTypes.bool
-  }
+    infinite: PropTypes.bool,
+    style: PropTypes.object
+  };
```

Comment:
My style is usually no semi-colons and then I'm forced to use them _only_ on static props because it breaks es6 syntax highlighting without them. According to [this](https://github.com/feross/standard/issues/372), I think Babel actually recommends you use them for static props.

I can take them out if you prefer.